support-frontend/assets/pages/subscriptions-landing/components/featuredProducts.jsx

```diff
@@ -168,6 +169,9 @@ const getProduct = (subsLinks: SubsUrls, countryGroupId: CountryGroupId): ?Produ
     case 'GuardianWeekly':
       return products.GuardianWeekly;
     default:
+      if (promoteGuardianWeekly) {
+        return products.GuardianWeekly;
+      }
```

Comment:
I know this isn't your change but there are 2 functionally identical if clauses here:
` if (countryGroupId === GBPCountries && flashSaleIsActive('DigitalPack', GBPCountries))`
and 
`if (countryGroupId !== GBPCountries && flashSaleIsActive('DigitalPack', countryGroupId))`
Could you remove one in this PR please?

Reply:
Sure thing